### PR TITLE
create load() and save() to pull data from localStorage and save data to localStorage

### PR DESCRIPTION
**Pull request by hnaht310**
- I finished task 8
- I added margin bottom to task card
- for task 7 code, I also made some minor changes relating to DOM traversing

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -420,22 +421,22 @@ input[type="text"] {
...
   -webkit-appearance: none;
   display: none;
 }
 
 /* Error Messages */
 
 .errorFonts {
-  color: #F8380E;
-  font-family: "Darker Grotesque", sans-serif;
+  color: #f8380e;
```
Comment (HenRMJ) [inline assets/css/style.css:438]: I prefer uppercase hex codes but I guess it's just preference

File: assets/js/index.js [hunk 2]
```diff
@@ -36,32 +40,40 @@ newTaskForm.addEventListener('submit', (event) => {
...
   }
 });
 
-
-
 taskList.addEventListener('click', (event) => {
   if (event.target.classList.contains('done')) {
-    let status = event.target.parentElement.parentElement.parentElement.parentElement.getElementsByTagName('ul')[0].getElementsByTagName('li')[0];
+    let status =
+      event.target.closest('.details').nextElementSibling.firstElementChild;
```
Comment (HenRMJ) [inline assets/js/index.js:62]: Yeah a lot simpler

**Review by HenRMJ — APPROVED** [review]
Nice looks good👍

File: assets/css/style.css (hunk 1 above)
Comment (hnaht310) [reply assets/css/style.css:438]: I didn't change it. It's prettier's fault again lol

**Pull request merged**